float64[] data
int32[] shape
